### PR TITLE
Consistent border radius across site

### PR DESCRIPTION
**Pull request by packaging-art**
Nothing in our mock-ups had a large border radius.

### CODE REVIEW

File: scss/_custom.scss [hunk 1]
```diff
@@ -227,9 +227,9 @@ $line-height-sm:         1.5 !default;
 
 $border-width: 1px !default;
 
-$border-radius:          .25rem !default;
-$border-radius-lg:       .3rem !default;
-$border-radius-sm:       .2rem !default;
+$border-radius:          1px;
```
Comment (anorborg) [inline scss/_custom.scss]: Can we stick to 'rem' units for consistency with the base?

**Review by anorborg — CHANGES_REQUESTED** [review]

**Comment by packaging-art** [comment]
good, call — agreed

**Review by anorborg — APPROVED** [review]
:shipit:

**Review by anorborg — APPROVED** [review]
still good. :shipit:

**Pull request merged**